src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -398,7 +408,9 @@ UCS_PROFILE_FUNC_ALWAYS(ucs_status_t, uct_ib_mlx5_devx_reg_atomic_key,
 
     status = uct_ib_mlx5_devx_reg_ksm_data_contig(
             md, mr, uct_ib_md_atomic_offset(mr_id), memh->address, is_atomic,
```

Comment:
pass iova (address + offset) instead of offset and address as 0 in case of memic